### PR TITLE
Fix test_variable_documentation failing test

### LINKED ISSUE #980: Failing Web API tests
Hi there!

I really enjoy OpenFisca, but I recently encountered an issue.

### Here is what I did:

```
make test
```

### Here is what I expected to happen:

```
508 passed, 1 xfailed, 32 warnings in 30.34s
```

### Here is what actually happened:

```
========================================================================================= short test summary info ==========================================================================================
FAILED tests/web_api/test_variables.py::test_variable_formula_content - assert 'def formula(...me_tax_rate\n' == 'def formula(...me_tax_rate\n'
FAILED tests/web_api/test_variables.py::test_variable_documentation - AssertionError: assert '\nHousing al... necessary.\n' == '\nTo compute... necessary.\n'
========================================================================== 2 failed, 506 passed, 1 xfailed, 32 warnings in 49.60s ==========================================================================
make: *** [test] Error 1
```

## Context

Some Web API tests rely on specifics of openfisca/country-template implementation, specifically literal strings - the objective is to be sure we can obtain de string literals of variables and their documentation via the Web API.

Or, as we might expect, that's fragile, as any change to the string representation of those elements will make the tests fail.

The present failing tests were introduced by openfisca/country-template#97

I identify more as a:

- Developer _(I create tools that use the existing OpenFisca code)_.

### PR DESCRIPTION
**Pull request by maukoquiroga**
#### Technical changes

- Fixes failing tests introduced by openfisca-core/country-template#97
- Fixes #980

### CODE REVIEW

File: tests/web_api/test_variables.py
Comment (Br3nda) [inline tests/web_api/test_variables.py:97]: could it be easier to read as:
```suggestion
    formula_code = """
    def formula(person, period, parameters):
        \"\"\"
        Income tax.

        The formula to compute the income tax for a given person at a given period
        \"\"\"
        return person("salary", period) * parameters(period).taxes.income_tax_rate
    """
```

Comment (Br3nda) [reply tests/web_api/test_variables.py:97]: ^^^ Suggestion only, I'll hit approve button

**Review by Br3nda — APPROVED** [review]

**Comment by MattiSG** [comment]
It would be great if this landed soon, all tests fail until this one is merged 🙂 For example, #982 is blocked because of that.

**Comment by MattiSG** [comment]
@maukoquiroga unless you'd prefer to finalise this, I intend to merge it today to unblock #982 🙂

**Comment by MattiSG** [comment]
This test could be made much more reliable by asserting the presence of a subset of the text rather than asserting full string equality.

**Comment by maukoquiroga** [comment]
Hi @MattiSG, thanks for the proposed PR, accepted it, way better !

**Pull request merged**
